2020/hamming-codes/hungarian/sentence_translations.json

```diff
@@ -973,7 +973,7 @@
  },
  {
   "input": "The eight bits are redundant in the sense that they're completely determined by the rest of the message, but it's in a much smarter way than simply copying the message as a whole.",
-  "translatedText": "A nyolc bit redundáns abban az értelemben, hogy teljesen az üzenet többi része határozza meg őket, de ez sokkal okosabb módon történik, mintha egyszerűen az üzenet egészét másolnánk.",
+  "translatedText": "Ez a nyolc bit persze redundáns abban az értelemben, hogy az üzenet többi része határozza meg őket, de ez sokkal okosabb módon történik, mintha az egész üzenetet lemásolnánk.",
```

Comment:
```suggestion
  "translatedText": "Ez a nyolc bit persze redundáns abban az értelemben, hogy az üzenet többi része határozza meg őket, de ez sokkal okosabb módszer, mint az egész üzenetet lemása.",
```

Reply:
I think this is a typo. "lemása" should be "lemásolása". I'll correct it in the file.